canonical/ubuntu.com
# K8s1.29+ck3 docs

## Done

- Updates various pages of K8s docs for 1.29+ck3 release

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
    
N.B. The content of these changes has already been reviewed
